src/service/navigation.js

```diff
@@ -40,6 +40,7 @@ const TAG = 'navigation';
 const EVENT_TYPE_CLICK = 'click';
 /** @private @const {string} */
 const EVENT_TYPE_CONTEXT_MENU = 'contextmenu';
+const VALID_TARGETS = ['_self', '_blank'];
```

Comment:
it's not really `_self` but rather `_top` that we support and is the default case. Let's update in the documentation above and here as well.